build-support/bin/release.sh

```diff
@@ -651,7 +683,7 @@ function usage() {
   echo " -p  Build a pex from prebuilt wheels for this release."
   echo " -q  Build a pex which only works on the host platform, using the code as exists on disk."
   echo
-  echo "All options (except for '-d') are mutually exclusive."
+  echo "All options (except for '-d' and '-3') are mutually exclusive."
```

Comment:
Good idea.

Even better will be using Python's `argparse` to handle this all for us + to allow us to use long-form arguments. Think it will become much more tenable when we drop Py2 and can start using `subprocess.run([my, bash, command])` over `subprocess.Popen().communicate()`.